### DISCUSSION TITLE
Empty error response in some plugin

### DISCUSSION THREAD

**Discussion by gikang82** [cloudforet-io/community#75] [Bug Report]
Hello I`m using SpaceONE service, but there is a empty error message from some collector

I`m using 'AWS Personal Health Dashboard Collector' there is a error message while executing collector

`No message from plugin`

Is there any possible reason to that message?
Usually plugin returns readable error message but it seems like a exceptional case

**Comment by jihyungSong** [comment]
I Think this case is a bug which messages are not delivered to Inventory Worker when an error occurs. 

I create an issue for this
* https://github.com/cloudforet-io/plugin-aws-phd-inven-collector/issues/4
